examples/nais.yaml

```diff
@@ -17,14 +18,14 @@ spec:
   replicas:
     min: 2
     max: 4
-  ingresses: 
+  ingresses:
```

Comment:
Litt på siden av denne saken, men hvordan skiller vi mellom on-prem og the cloud? For istio blir vel ikke rullet ut on-prem :/

Reply:
Onprem er ikke oppstartsflagget flagget `accessPolicy` satt til true. Da vil ikke naiserator generere istio- eller networkpolicy-ressurser. Hvis det er `accessPolicy` i nais.yaml vil dette bare bli ignorert.